src/system_time.rs

```diff
@@ -0,0 +1,27 @@
+use std::time::Duration;
+
+#[cfg(target_arch = "wasm32")]
+pub(crate) fn now() -> f64 {
+    let now = {
+        use wasm_bindgen_rs::prelude::*;
+        js_sys::Reflect::get(&js_sys::global(), &JsValue::from_str("performance"))
+            .expect("performance object not found")
+            .unchecked_into::<web_sys::Performance>()
+            .now()
+    };
+    now
+}
+
+#[cfg(not(target_arch = "wasm32"))]
+pub(crate) fn now() -> f64 {
+    std::time::SystemTime::now().duration_since(std::time::SystemTime::UNIX_EPOCH)
+                            .expect("System clock was before 1970.")
+                            .as_secs_f64() * 1000.0
```

Comment:
```suggestion
                            .as_secs_f64() * Unit::Second
```

By using the Unit enum, the `now()` function would directly return a `Duration`, so the function could be directly `duration_since_unix_epoch()` I think.

Reply:
Great idea! I have refactored `system_time:: duration_since_unix_epoch()` to directly return hifitime::Duration